foreshadow/tests/test_transformers/test_transformers.py

```diff
@@ -571,3 +571,186 @@ def test_sparse_matrix_conversion():
     # This tf generates sparse output by default and if not handled will
     # break pandas wrapper
     tfidf.fit_transform(corpus)
+
+
+@pytest.mark.parametrize(
+    "transformer,input_csv",
+    [
+        ("StandardScaler", "./foreshadow/tests/test_data/boston_housing.csv"),
```

Comment:
These paths should probably use the same style of relative file sourcing that was added in the develepment branch through the darglint pr

Reply:
Will do